### PR TITLE
feat(provider): Add Sendchamp SMS provider integration

### LINKED ISSUE #3637: 🚀 Feature: Add Sendchamp provider
### 🔖 Feature description

New [Sendchamp](https://www.sendchamp.com/) SMS Provider.

### 🎤 Why is this feature needed ?

To have the ability to use [Sendchamp](https://www.sendchamp.com/) as a provider.

### ✌️ How do you aim to achieve this?

Add a new provider based on [this guide](https://docs.novu.co/community/create-provider)

### 🔄️ Additional Information

_No response_

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by peoray**
### What change does this PR introduce?
Add Sendchamp SMS provider integration

### Why was this change needed?

Closes #3637

### CODE REVIEW

File: apps/web/public/static/images/providers/dark/sendchamp.svg
Comment (BiswaViraj) [inline apps/web/public/static/images/providers/dark/sendchamp.svg:1]: I think this won't look good in dark mode.
Can you please replace it with this?
https://res.cloudinary.com/sendchamp/image/upload/v1647472665/Sendchamp%20Website/logos/SendchampWhite_kelitk.svg

Comment (unicodeveloper) [reply apps/web/public/static/images/providers/dark/sendchamp.svg:1]: @peoray

Comment (peoray) [reply apps/web/public/static/images/providers/dark/sendchamp.svg:1]: @unicodeveloper @BiswaViraj done

**Comment by unicodeveloper** [comment]
@peoray Please can you resolve the conflicts?

**Comment by jainpawan21** [comment]
@peoray 
Thanks for your contribution 🎉
Can you please add step-by-step documentation also for this new provider?

**Comment by peoray** [comment]
@unicodeveloper all done

**Comment by BiswaViraj** [comment]
@peoray Can you please resolve the conflicts?

**Review by BiswaViraj — CHANGES_REQUESTED** [review]
@peoray Awesome stuff 🌟
Just a small change needed
Can you please add `Sendchamp` and `sendchamp` to this [file](https://github.com/novuhq/novu/blob/ac2a69d2fba5a5bc499b8e5d56008743f1b160cb/.cspell.json#L490)
So that it will fix the spelling error CI test

**Review by BiswaViraj — CHANGES_REQUESTED** [review]
Also can you please add documentation for it over [here](https://github.com/novuhq/novu/tree/next/docs/docs/channels/sms)

**Comment by peoray** [comment]
@BiswaViraj documentation exists in another PR (#3719)

**Comment by peoray** [comment]
@BiswaViraj done :)

**Review by BiswaViraj — APPROVED** [review]
🌟

**Pull request merged**